app/controllers/bookingsync_portal/admin/rentals_controller.rb

```diff
@@ -5,19 +5,44 @@ class RentalsController < Admin::BaseController
       before_action :fetch_remote_rentals, only: :index
 
       def index
-        @not_connected_rentals = current_account.rentals.visible.ordered.not_connected
-        @visible_rentals = current_account.rentals.visible
-        @remote_accounts = current_account.remote_accounts
-        @remote_rentals_by_account = current_account.remote_rentals.ordered
-          .includes(:remote_account, :rental).group_by(&:remote_account)
+        render :index, locals: { **index_arguments }
       end
 
       def show
         rental
       end
 
+      def index_arguments
+        BookingsyncPortal.custom_arguments.dig("Admin", "RentalsController", "index") || {
+          not_connected_rentals: not_connected_rentals.call(current_account),
+          visible_rentals: visible_rentals.call(current_account),
+          remote_accounts: remote_accounts.call(current_account),
+          remote_rentals_by_account: remote_rentals_by_account.call(current_account)
+        }
+      end
+
       private
 
+      def not_connected_rentals
+        BookingsyncPortal.not_connected_rentals || lambda {
```

Comment:
Hmm we decided to override full action, therefore customizations for each ivar are not needed. We want to overridde whole action